scripts/build-wasm.sh

```
@@ -176,5 +176,44 @@ if [ "${BUILD_THREADED:-}" = "1" ]; then
+  WIDE_OUT="../../packages/wasm/pkg-wide"
+  echo ""
+  echo "🔢 Building wide-arithmetic bundle → $WIDE_OUT (+wide-arithmetic)"
+  # RUSTFLAGS fully replaces .cargo/config.toml's target rustflags, so repeat the
+  # base flags (max-memory, stack-size, simd128) and add +wide-arithmetic.
+  RUSTFLAGS="-C link-arg=--max-memory=4294967296 -C link-arg=-zstack-size=8388608 -C target-feature=+simd128,+wide-arithmetic" \
+  rustup run nightly-2025-11-15 "$WASM_PACK" build rust/wasm-bindings \
+    --target web \
+    --out-dir "$WIDE_OUT" \
+    --out-name ifc-lite \
+    --release \
+    $FEATURES
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)